eng/common/core-templates/steps/get-delegation-sas.yml

```diff
@@ -28,15 +28,20 @@ steps:
     scriptType: 'pscore'
     scriptLocation: 'inlineScript'
     inlineScript: |
-      # Calculate the expiration of the SAS token and convert to UTC
-      $expiry = (Get-Date).AddHours(${{ parameters.expiryInHours }}).ToUniversalTime().ToString("yyyy-MM-ddTHH:mm:ssZ")
+      # Temporarily work around a helix issue where SAS tokens with / in them will cause incorrect downloads
+      # of correlation payloads. See https://github.com/dotnet/dnceng/issues/3484
+      $sas = ""
+      do {
+        # Calculate the expiration of the SAS token and convert to UTC
+        $expiry = (Get-Date).AddHours(${{ parameters.expiryInHours }}).ToUniversalTime().ToString("yyyy-MM-ddTHH:mm:ssZ")
```

Comment:
Did you want to subtract a minute from Get-Date? We found that sas tokens can sometimes fail due to clock skew when using 'now' operations for start time validity

Reply:
In this case you are only setting up the expiry so start time should make no difference.